src/content/docs/apm/new-relic-apm/apdex/apdex-measure-user-satisfaction.mdx

```diff
@@ -25,76 +25,89 @@ Setting useful thresholds for your Apdex levels help you:
+  />
+
+  <TechTile
+    name="Python agent"
+    icon="logo-python"
+    to="https://one.newrelic.com/nr1-core?state=20fda75b-58fb-a92a-f9e1-7b052035c6e8"
+  />
+
+  <TechTile
+    name="Ruby agent"
+    icon="logo-ruby"
+    to="https://one.newrelic.com/nr1-core?state=d69143ab-605c-579b-25bf-cc6e5fee5b80"
+  />
+
+</TechTileGrid>
+
 To learn more, watch this short YouTube video (approx. 3:50 minutes).
 
 <Video
   id="VqOLmXxpkTw"
   type="youtube"
 />
 
-Ready to get started? If you haven't already, be sure to [sign up for a New Relic account](https://newrelic.com/signup). It's free, forever.
-
-## Apdex measurements [#what-is]
-
-Apdex is a measure of response time based against a set threshold. It measures the ratio of satisfactory response times to unsatisfactory response times. The response time is measured from an asset request to completed delivery back to the requestor.
-
-After you define a response time threshold `T`, all responses handled in `T` or less time satisfy the user.
+## 
 
-For example, if `T` is 1.2 seconds and a response completes in 0.5 seconds, then the user is satisfied. All responses greater than 1.2 seconds dissatisfy the user. Responses greater than 4.8 seconds frustrate the user.
+## How to use Apdex [#what-is]
 
-You can define multiple Apdex T values for each of these:
-
-<table>
-  <thead>
-    <tr>
-      <th style={{ width: "200px" }}>
-        Apdex measurements
-      </th>
-
-      <th>
-        Comments
-      </th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <tr>
-      <td>
-        APM apps
-      </td>
+The Apdex score is a ratio value of the number of satisfied and tolerating requests to the total requests made. Each satisfied request counts as one request, while each tolerating request counts as half a satisfied request.
 
-      <td>
-        APM monitors the performance of these apps. To [define their Apdex settings](/docs/apm/new-relic-apm/apdex/change-your-apdex-settings), use the APM UI. You can find **Apdex** on the [APM Summary](/docs/apm/applications-menu/monitoring/apm-overview-page) page.
-      </td>
-    </tr>
+An Apdex score varies from 0 to 1, with 0 as the worst possible score (100% of response times were Frustrated), and 1 as the best possible score (100% of response times were Satisfied).
 
-    <tr>
-      <td>
-        Browser apps
-      </td>
+**Example Apdex score:**
 
-      <td>
-        Browser monitors the end-user experience for these apps. Instead of using Apdex to monitor your users, we recommend [core web vitals](/docs/browser/browser-monitoring/getting-started/browser-summary-page/#core_vitals).
-      </td>
-    </tr>
+During a 2 minute period a host handles 200 requests. The Apdex threshold T = 0.5 seconds (500ms). This value is arbitrary and is selected by the user.
 
-    <tr>
-      <td>
-        Key transactions
-      </td>
+* 170 of the requests were handled within 500ms, so they are classified as Satisfied.
```

Comment:
they're :). Also for my edification, is there any special formatting for the classifications in these bullets?